web-server/env-vars

```diff
@@ -0,0 +1,22 @@
+export TF_VAR_region="eu-frankfurt-1"
```

Comment:
This file shouldn't be in Git - it contains potentially dangerous information, e.g. a private key password etc.